### PR TITLE
Documentation: Nullannotations should be written before the type

### PR DESCRIPTION
**Pull request by triller-telekom**
Signed-off-by: Stefan Triller <stefan.triller@telekom.de>

### CODE REVIEW

File: docs/documentation/development/conventions.md [hunk 1]
```diff
@@ -18,11 +18,24 @@ For private methods or methods in an internal package the annotations are respec
 
 To use the annotations, every bundle must have an **optional** `Import-Package` dependency to `org.eclipse.jdt.annotation`.
 Classes should be annotated by `@NonNullByDefault` and return types, parameter types, generic types etc. are annotated with `@Nullable` only.
+The annotation should be written before the type, not before the visibility modifier.
```
Comment (htreu) [inline docs/documentation/development/conventions.md]: I propose to change "before" to "in front of".

Comment (hsudbrock) [reply docs/documentation/development/conventions.md]: Should this only be done for field declarations (like, e.g., `private @Nullable String bla;`)? Or also on the class level (like, e.g., `public @NonNullByDefault class Blubb`)?

Comment (maggu2810) [reply docs/documentation/development/conventions.md]: I agree

**Comment by hsudbrock** [comment]
Should this only be done for field declarations (like, e.g., `private @Nullable String bla;`)? Or also on the class level (like, e.g., `public @NonNullByDefault class Blubb`)?

**Comment by maggu2810** [comment]
> Should this only be done for field declarations (like, e.g., `private @Nullable String bla;`)? Or also on the class level (like, e.g., `public @NonNullByDefault class Blubb`)?

IMHO for the class it could be kept as it is.
I don't know any class annotation that is written between the access modifier and the class attribute.

**Comment by triller-telekom** [comment]
> IMHO for the class it could be kept as it is.
I don't know any class annotation that is written between the access modifier and the class attribute.

I agree and add a sentence about the class annotation

**Comment by hsudbrock** [comment]
> IMHO for the class it could be kept as it is.
I don't know any class annotation that is written between the access modifier and the class attribute.

I agree; I think, however, that this difference is not clear from the documentation currently.

**Comment by triller-telekom** [comment]
I have changed the "before" to "in fornt of" and also added and example on how to annotate a class.

**Pull request merged**